core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -169,11 +169,14 @@ abstract class TaskContext extends Serializable {
   /**
    * Return the local values of accumulators that belong to this task. The key of the Map is the
    * accumulator id and the value of the Map is the latest accumulator local value.
+   * @param includeConsistent if we should include consistent accumulators.
+   * If we didn't process the full partition should be false.
    */
-  private[spark] def collectAccumulators(): Map[Long, Any]
+  private[spark] def collectAccumulators(includeConsistent: Boolean): Map[Long, Any]
```

Comment:
Does that mean that the program is multi-threading and during rdds being calculated you can collect accumulators ?
I think if Consistent accumulator has been collected once, it's value can't change after that